Jenkinsfile

```diff
@@ -0,0 +1,576 @@
+// Copyright (c) 2017, 2022, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+//
+def kind_k8s_map = [
+     '0.8': [
```

Comment:
I would be okay with eliminating all of the data for pre-0.11.1 Kind. We're not going to ever go back to those releases.

Reply:
Obviously, we can.  The question is does the 3.3 or 3.4 branches need earlier versions?

Reply:
No, they don't. We've been running all branches with the modern Kind binary for a while.